### ISSUE TITLE
Adopt Spectral provided Zalando rules

### ISSUE THREAD

**Issue by dweber019** [baloise-incubator/spectral-ruleset#41]
Replace own rules with the ones coming from Spectral.
https://apistylebook.stoplight.io/docs/zalando-restful-api-guidelines

**Comment by christiansiegel** [comment]
Looks like a fork without attribution. The sportlight ruleset contains 1:1 copies of some of the [functions](https://github.com/baloise-incubator/spectral-ruleset/tree/main/functions).

**Comment by dweber019** [comment]
True. Even Baloise is there now https://apistylebook.stoplight.io/docs/baloise 😉
